src/fusion-rates-gamow.md

```diff
@@ -0,0 +1,861 @@
+
+```python
+r_start = 74000
+gamow_74 = []
+
+for state in states:
+    V_eff = V_nuc(r_fm, state["V0"], state["r_S"], state["a_S"]) + V_cent(r_fm, state["L"]) + V_mol(r_fm)
+    G = calculate_G(r_fm, V_eff, r_start)
+    exp2G = np.exp(-2*G)
+    gamow_74.append({
+        "state": state['state'],
+        "$G$": G,
+        "$e^{-2G}$": exp2G
+    })
+
+# Convert the list of dictionaries to a DataFrame
+gamow_74 = pd.DataFrame(gamow_74)
+
+# Display the DataFrame
+gamow_74
+```
+
+In order to calculate the fusion rate $\Gamma$, we now need to come back calculate the volume factor and the the nuclear relaxation/decay rate $\gamma$ in:
+
+$$P = \frac{v_{nuc}}{v_{mol}}e^{-2 G}\gamma$$
+
+
+## Volume factor for $D_2$
+
+
+The ratio of the nuclear volume to molecular volume, ${v_{nuc} / v_{mol}}$, depends slightly on the state of the $\rm D_2$ molecules. To estimate the ratio, we begin by defining the equilibrium bond lengths for different electronic states of a $ \text{D}_2 $ molecule in terms of the Bohr radius ($ a_0 $):
+
+- For the S state: $ R_{0S} = 1.401080 \times a_0 $
+- For the P state: $ R_{0P} = R_{0S} + 0.001 \times 10^{-10} \, \text{m} $
+- For the D state: $ R_{0D} = R_{0S} + 0.002 \times 10^{-10} \, \text{m} $
+
+Next, we calculate the de Broglie wavelength of the relative motion ($ \Delta R $) using the formula:
+
+$$
+\Delta R = a_0 \sqrt{\frac{I_H}{\hbar \omega_{0DD}}} \sqrt{\frac{2 m_e c^2}{M_{D} c^2}}
+$$
+
+where $ I_H $ is the ionization potential of hydrogen in eV, $ \hbar \omega_{0DD} $ is the zero-point energy (0.3862729 eV), $ m_e c^2 $ is the electron mass in MeV, and $ M_D c^2 $ is the deuteron mass in MeV.
+
+The molecular volumes ($ v_{mol} $) for the S, P, and D states are computed as follows:
+
+- S: $ v_{mol} = 2 \pi R_{0S} \left(\pi \Delta R^2\right) $
+- P: $ v_{mol} = 2 \pi R_{0P} \left(\pi \Delta R^2\right) $
+- D: $ v_{mol} = 2 \pi R_{0D} \left(\pi \Delta R^2\right) $
+
+The nuclear volume ($ v_{nuc} $) of the deuterium nucleus is calculated assuming a spherical shape with a radius of 5 fm:
+
+$$
+v_{nuc} = \frac{4}{3} \pi \left(5 \times 10^{-15} \, \text{m}\right)^3
+$$
+
+Finally, the volume ratios of the nuclear volume to the molecular volume for the S, P, and D states are calculated as:
+
+- S: $ \frac{v_{nuc}}{v_{mol}} = 6.66 \times 10^{-12} $
+- P: $ \frac{v_{nuc}}{v_{mol}} = 6.65 \times 10^{-12} $
+- D: $ \frac{v_{nuc}}{v_{mol}} = 6.64 \times 10^{-12} $
+
+```python
+for i, ratio in enumerate([6.66e-12, 6.66e-12, 6.65e-12, 6.64e-12]):
+    states[i]["$v_{nuc}/v_{mol}$"] = ratio
+    gamow_74.loc[i,"$v_{nuc}/v_{mol}$"] = ratio
+
+gamow_74
+```
+
+## Nuclear relaxation/decay for $\rm D_2$ fusion
+
+
+When the deuterons tunnel through the Coulomb barrier, for a very brief moment, they form a highly clustered state that can be recognised as an excited state of $\rm ^4He$ (often denoted $\rm ^4He^*$). There are several excited states and they decay via different "channels". For example:
```

Comment:
Is it really highly clustered? I sometimes used this language "highly clustered" to describe the D2 molecule configuration but I realize that is more like "extremely clustered" and there are certain highly clustered nuclear states. I'm just not sure whether that here is one of them. Again, maybe "highly excited" is more neutral/agnostic as to the shape?